### DISCUSSION TITLE
Is there a way to get more information from AutoKeras?

### DISCUSSION THREAD

**Discussion by Rolun** [keras-team/autokeras#1477] [Q&A]
Hi,

I was wondering if there is a way to fetch all the models which AutoKeras tries, or if it's only possible to get the best one?

Also, what kind of metrics are possible to get during the training? Is it for example possible to create custom metrics like AUC for example?

Thanks in advance! :)

**Comment by haifeng-jin** [comment]
The `auto_model.tuner` will give you a Keras Tuner's `Tuner` object. You can check here on how to get more info https://github.com/keras-team/keras-tuner
You need to pass the custom metrics to the head, for example, `ClassificationHead(metrics=[custom_metric])`
https://autokeras.com/block/#classificationhead
